main.go

```diff
@@ -0,0 +1,5 @@
+package main
+
+func main() {
+
```

Comment:
Maybe add something like printing `Welcome to Shipwright Triggers` to stdout.

Reply:
Sure!